tests/lms/views/error_test.py

```diff
@@ -58,6 +58,22 @@ def test_it_sets_correct_status_int_for_non_http_error(self, pyramid_request):
 
         assert pyramid_request.response.status_int == 500
 
+    def test_it_sets_correct_message_for_missing_lti_param_error(self, pyramid_request):
+        exc = MissingLTILaunchParamError('test lti launch param error msg')
+
+        error_views = error.ErrorViews(exc, pyramid_request)
+        resp = error_views.missing_lti_launch_param_error()
+
+        assert resp['message'] == 'test lti launch param error msg'
+
+    def test_it_sets_correct_status_int_for_missing_lti_param_error(self, pyramid_request):
+        exc = MissingLTILaunchParamError('test lti launch param error msg')
+
+        error_views = error.ErrorViews(exc, pyramid_request)
+        resp = error_views.missing_lti_launch_param_error()
+
+        assert pyramid_request.response.status_int == 400
+
```

Comment:
I don't understand why these new tests are in this PR? This PR doesn't change `error.py`